plugins/BEdita/API/src/Controller/ObjectTypesController.php

```diff
@@ -13,7 +13,7 @@
 namespace BEdita\API\Controller;
 
 /**
- * Controller for `/object_types` endpoint.
+ * Controller for `/model/model/object_types` endpoint.
```

Comment:
should it be instead `/model/object_types`?

Reply:
of course, it's a typo...